src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -108,6 +108,6 @@ public static class ScriptConstants
         /// This constant is also defined in Antares, where the limit is ultimately enforced
         /// for settriggers calls. If we raise that limit there, we should raise here as well.
         /// </summary>
-        public const int MaxTriggersStringLength = 102400;
```

Comment:
As part of the investigations I was doing, I noticed that this value is out of sync with dev https://github.com/Azure/azure-functions-host/blob/e94fb499e791847b6f03c9bd1f81335f3d0a5905/src/WebJobs.Script/ScriptConstants.cs#L123

Reply:
This commit https://github.com/Azure/azure-functions-host/commit/204d16c9bcde05eb896f55f0007a0b273627b90a was apparently never merged to v1